Platform/CommonBoardPkg/Library/BoardSupportLib/BoardSupportLib.c

```diff
@@ -189,7 +189,7 @@ SpiLoadExternalConfigData (
 
   SignedLen = CfgBlob->UsedLength;
   if (FeaturePcdGet (PcdVerifiedBootEnabled)) {
-    SignedLen += RSA_SIGNATURE_AND_KEY_SIZE;
+    SignedLen += SIGNATURE_AND_KEY_SIZE_MAX;  //Allocate maximum key and signature size available

```

Comment:
Use the max size (RSA2072) for signedlen to be allocated.
It could be improved by adding Sign/hash type params to CFG_BLOB.

Reply:
This will cause extra flash to be read and copied.
We should use the exact size instead.  I think this could be decided by a PCD or check the PUB_KEY_HDR follow the UsedLength.

Reply:
This size is dynamic based on the private passed while signing. We do want to add PCD for mentioning key sizes from build.
For getting info through Signature and PubKey Hdr, multiple flash reads would be required.
I would take this approach as separate patch. I would revert for RSA2K required size for this patch,